modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase

```diff
@@ -154,6 +154,342 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10503. Minimum quantity can be added to cart when order is higher then stock quantity"
+	@priority = "3"
+	test CanAddedMinimumQuantityWhenOrderIsHigherThenStock{
+		property portal.acceptance = "false";
+
+		task ("Given a buyer user is created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And given the warehouse quantity for the product is set to 6") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceEntry.editWarehousesQuantity(
+				italyQuantity = "2",
+				northeastQuantity = "2",
+				productName = "U-Joint",
+				productSku = "MIN55861",
+				southwestQuantity = "2");
+		}
+
+		task ("And minimum quantity is set to 8 and Allow Back Order is abled") {
+			CommerceProducts.openProductsAdmin();
+			
+			CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+			CommerceNavigator.gotoEntry(entryName = "U-Joint");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Configuration");
+			
+			CommerceProducts.updateProductInventoryConfiguration(
+				allowBackOrders = "true",
+				displayAvailability = "true",
+				displayStockQuantity = "true",
+				minOrderQuantity = "8",
+				publish = "true");
+		}
+
+		task ("And buyer user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("When buyer user add minimum quantity to the cart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "8");
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+		
+		task ("Then buyer user can do the checkout with the quantity higher then the stock quantity "){
```

Comment:
task ("Then buyer user can do the checkout with the quantity higher than the stock quantity") {